pypreprocess/nipype_preproc_spm_utils.py

```diff
@@ -379,14 +383,23 @@ def _do_subject_realign(subject_data, reslice=False, register_to_mean=False,
     if report:
         subject_data.generate_realignment_thumbnails()
 
+    if report_path not in [False,None]:
+        rp_plot, rp_log = generate_realignment_report(
+            subject_data=subject_data,
+            estimated_motion=subject_data.realignment_parameters,
+            output_dir=subject_data.output_dir,
+            report_path=report_path
+            )
+        add_component(rp_plot,report_path,rp_log,log_path)
+
     return subject_data.sanitize()
 
 
 def _do_subject_coregister(subject_data, reslice=False, spm_dir=None,
                            matlab_exec=None, spm_mcr=None,
                            coreg_anat_to_func=False, caching=True,
-                           report=True, software="spm", hardlink_output=True,
-                           **kwargs):
+                           report=True, software="spm", hardlink_output=True
+                           ,report_path=None,log_path=None,**kwargs):
```

Comment:
the comma should be at the end of the line
```suggestion
                           , report_path=None, log_path=None, **kwargs):
```